### DISCUSSION TITLE
pnpm install did nothing but show Scope: all 7 workspace projects

### DISCUSSION THREAD

**Discussion by selma-li** [pnpm/pnpm#4264] [Q&A] [answered]
pnpm install did nothing but show Scope: all 7 workspace projects.
```
xxxxxxxxx xxxxx %pnpm install
Scope: all 7 workspace projects
xxxxxxxxx xxxxx %pnpm --version
6.24.2
```
device: using macOS Big Sur
I will appreciate if anyone could answer me.

**Comment by zkochan** [comment]
Do you have any dependencies in your projects?

**Reply by selma-li** [reply]
Thank you. I found the solution. I should write all the workspaces in the `pnpm-workspace.yaml`

**Reply by zxcvc** [reply]
> Do you have any dependencies in your projects?

same problem in 8.2.0
have some dependencies.
but I dont want to write all the workspaces in the pnpm-workspace.yaml,how to solve?

**Comment by nullice** [comment]
same problem in v7.27.0    2023

**Comment by selma-li** [comment, marked as answer]
I found the solution. I should write all the workspaces in the pnpm-workspace.yaml.

**Reply by bChandrax** [reply]
Thank you so much
